server/chat-plugins/the-studio.ts

```diff
@@ -0,0 +1,613 @@
+import {FS} from '../../lib/fs';
+import {Net} from '../../lib/net';
+import {Utils} from '../../lib/utils';
+import {YoutubeInterface} from './youtube';
+
+const YouTube = new YoutubeInterface();
```

Comment:
Is there a reason to construct a new one rather than just using the exported instance?

Reply:
Might as well just import the interface so i can use the static function, and theres no harm in not using the exported class from the file since i dont need the channel interfaces